### ISSUE TITLE
CSS: Use ids-identity typography tokens

### ISSUE THREAD

**Issue by nickwynja** [infor-design/enterprise#1640]
**Is your feature request related to a problem? Please describe.**
Typography in EP is not currently tied to the tokens.

**Describe the solution you'd like**
Use the typography related tokens in the core styles so that we see changes in the type styles when using the uplift theme. 

Since `@include font-size()` expects a digit and then converts to both `px` and `rem`, we'll need to just remove than in these new cases.

Then, we could do a global find/replace on the following:

- [x] `@include font-size(10)` => `font-size: $theme-size-font-xs;`
- [x] `@include font-size(12)` => `font-size: $theme-size-font-sm;`
- [x] `@include font-size(14)` => `font-size: $theme-size-font-base;`
- [x] `@include font-size(18)` => `font-size: $theme-size-font-lg;`
- [x] `@include font-size(24)` => `font-size: $theme-size-font-xl;`

We'd also have to update the current `theme-soho/theme.json` tokens to something like this that better aligns with the current font values (https://github.com/infor-design/design-system/issues/295)

```
"xs":   { "value": "1.0rem" },
"sm":   { "value": "1.2rem" },
"md":   { "value": "1.4rem" },
"lg":   { "value": "1.8rem" },
"xl":   { "value": "2.4rem" },
```

**Describe alternatives you've considered**
We probably will want to swap out certain component size values with the specific component token but this will help get things started.

**Additional context**
n/a

**Comment by clepore** [comment]
Here are the remaining "odd" fonts randomly throughout Enterprise...
<img width="301" alt="screen shot 2019-03-06 at 12 55 09 pm" src="https://user-images.githubusercontent.com/742319/53902518-4f432b80-400f-11e9-9295-f3eb05faf432.png">

**Comment by tmcconechy** [comment]
The `font-size: 0` is important as it will remove whitespace on certain elements. If you remove some of these it will definitely break some things. We use this technique:

https://css-tricks.com/fighting-the-space-between-inline-block-elements/#article-header-id-3

**Comment by tmcconechy** [comment]
Also i think app/src/docs.scss and anything around it can be removed. This is for the old docs site.

**Comment by clepore** [comment]
That leaves just the `em`'s left then and I can't really convert those to `rem` without going through them one by one:
<img width="431" alt="screen shot 2019-03-06 at 1 19 47 pm" src="https://user-images.githubusercontent.com/742319/53903782-967eeb80-4012-11e9-96ea-eec436c30ac9.png">

**Comment by nickwynja** [comment]
Closed by #1751.
